lib/pg_ha_migrations/unsafe_statements.rb

```diff
@@ -4,7 +4,7 @@ def self.delegate_unsafe_method_to_connection(method_name)
       arg_list = args.map { |arg| arg.inspect }.join(', ')
       # say_with_time args taken from https://github.com/rails/rails/blob/4-2-stable/activerecord/lib/active_record/migration.rb#L654
       say_with_time "#{method_name}(#{arg_list})" do
```

Comment:
We might be able to avoid the need for `say_with_time` if we are calling the migration class instead of the connection?